src/rai/rai/node.py

```diff
@@ -109,7 +109,8 @@ def append_whoami_info_to_prompt(
         prompt=prompt,
     )
 
-    node.get_logger().info(f"System prompt initialized: {system_prompt}")
+    node.get_logger().info("System prompt initialized")
+    node.get_logger().debug(system_prompt)
```

Comment:
Add some explanation to the log
```suggestion
    node.get_logger().debug(f"System prompt:\n{system_prompt}")
```